smst.go

```diff
@@ -71,30 +77,42 @@ func (smst *SMST) Spec() *TrieSpec {
 	return &smst.TrieSpec
 }
 
-// Get retrieves the value digest for the given key and the digest of the value
-// along with its weight provided a leaf node exists.
-func (smst *SMST) Get(key []byte) (valueDigest []byte, weight uint64, err error) {
+// Get retrieves the value digest for the given key, along with its weight and count
+// of non-empty leafs assuming the node exists.
+func (smst *SMST) Get(key []byte) (valueDigest []byte, weight, count uint64, err error) {
```

Comment:
Would we break anything if we omit returning `count` from `smst.Get` ? I believe it would be more concise API if this was not returning `count`